Header header
robot_manipulation/GraspCandidate[] candidates
